docs/cli/cozy-stack_fixer.md

```diff
@@ -32,6 +32,7 @@ cozy-stack fixer <command> [flags]
 * [cozy-stack fixer albums-created-at](cozy-stack_fixer_albums-created-at.md)	 - Add a created_at field for albums where it's missing
 * [cozy-stack fixer contact-emails](cozy-stack_fixer_contact-emails.md)	 - Detect and try to fix invalid emails on contacts
 * [cozy-stack fixer jobs](cozy-stack_fixer_jobs.md)	 - Take a look at the consistency of the jobs
+* [cozy-stack fixer link-app](cozy-stack_fixer_link-app.md)	 - Link an old OAuth client to a webapp
```

Comment:
I don't think this line is related to this PR, isn't it?

Reply:
I confirm: it was just some documentation missing from a previous PR